### DISCUSSION TITLE
Display changes in a vsplit buffer like `Enter` a commit in `LogView`

### DISCUSSION THREAD

**Discussion by LintaoAmons** [NeogitOrg/neogit#626] [Q&A] [answered]
In `NeogitStatus` buffer, I can hit `<Tab>` to view the changes in this format

![image](https://github.com/NeogitOrg/neogit/assets/95092244/b8ac6c1e-e0fa-4708-9545-33048e8660ff)

In `NeogitLogView`, I can hit `<CR>` to view the changes made of the commit under the cursor.

![image](https://github.com/NeogitOrg/neogit/assets/95092244/8c2d9ddc-ecf8-468b-a930-b26404586795)

I wonder if it's possible to have the `NeogitStatus` have this split view, easier to see more information

**Comment by CKolkey** [comment, marked as answer]
So what you're seeing on the right in the `vsplit` arrangement is the `commit_view` - that's only for a commit that already exists. The status buffer, by contrast, shows you the working index as it changes. There's no plan at this time to change how the status buffer works.

**Reply by LintaoAmons** [reply]
Ohhh, I got it~ Thx for your explain
